interpreter/code_interpreters/languages/python.py

```diff
@@ -53,11 +55,23 @@ def preprocess_python(code):
     return code
 
 
+def add_last_line_print(tree):
```

Comment:
I had problems running the code in interactive mode and getting output as you do, so I added this workaround. It adds print on the last statement. If I won't find a way to run it the same way as you do, I will refactor this little bit